mlrun/model_monitoring/db/stores/sqldb/models/__init__.py

```diff
@@ -25,3 +29,21 @@ def get_model_endpoints_table(
     if connection_string and "mysql:" in connection_string:
         return MySQLModelEndpointsTable
     return SQLiteModelEndpointsTable
+
+
+def get_application_result_table(
+    connection_string: Optional[str] = None,
+) -> Union[type[MySQLApplicationResultTable], type[SQLiteApplicationResultTable]]:
+    """Return ModelEndpointsTable based on the provided connection string"""
+    if connection_string and "mysql:" in connection_string:
+        return MySQLApplicationResultTable
+    return SQLiteApplicationResultTable
```

Comment:
```suggestion
    connection_string = connection_string or mlrun.model_monitoring.helpers.get_connection_string(
                secret_provider=secret_provider
            )
    if connection_string and "mysql:" in connection_string:
        return MySQLApplicationResultTable
    return SQLiteApplicationResultTable
```

Reply:
as we discussed, we removed the connection_string argument from the sql_store + update the get tables functions to be a private functions.